practice-one/src/router/Routers.css

```diff
@@ -0,0 +1,4 @@
+.main-content {
+    margin-left: 218px;
+    padding: 30px 30px 0 30px;
+}
```

Comment:
???

Reply:
I use margin-left to align main-content because I used position fixed for sidebar 
If I don't use margin-left, the main-content will be covered by sidebar
![image](https://github.com/datdragon777/React-Training/assets/74806826/09527e2e-217f-43f3-944b-aab5d9d75d49)